src/Illuminate/Http/Middleware/VerifyEnvironment.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Http\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Foundation\Application;
+
+class VerifyEnvironment
```

Comment:
My personal opinion but I'd use `EnvironmentVerifier` as the name of this class

Reply:
The names of middleware are usually English readable actions.

"Check response for modifications", "Handle CORS", "Set cache headers", etc
See: https://github.com/laravel/framework/tree/9.x/src/Illuminate/Http/Middleware

Reply:
Oh yeah, I totally missed that, thanks! :sweat_smile: